content-src/components/DiscoveryStreamComponents/Navigation/_Navigation.scss

```diff
@@ -0,0 +1,44 @@
+.ds-navigation {
+
+  &.ds-navigation-centered {
+    text-align: center;
+  }
+
+  &.ds-navigation-right-aligned {
+    text-align: right;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  ul li {
+    display: inline-block;
+
+    &::after {
+      content: '•';
+      padding: 8px;
+    }
+
+    &:last-child::after {
+      content: none;
+    }
+
+    a {
+      color: $blue-60;
+    }
+  }
+
+  .ds-more-recommendations::after {
```

Comment:
Is it expected to use the `&::after` related declarations for this too?

Reply:
Not sure I'm totally following.

Reply:
Multiple selectors would have applied styles to the "more" li::after, but we're getting rid of this, so nothing special anymore.